### PR TITLE
add a simple program to access the pvt properties of co2 and brine

### PR DESCRIPTION
**Pull request by totto82**
A simple program that can be used to compute pvt properties of the co2-brine system
```sh
co2brinepvt density CO2 1e6 300
```
should give you the density of CO2 at 1e6pa and 300K. 

Useful for instance if you want to generate blackoil tables for co2brine

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by blattms** [comment]
Cool.

I guess this program is supposed to be installed for general usage? In that case please add a comprehensive help page (the we can use that to create a man-page) and make sure it gets installed to the right place.

**Comment by totto82** [comment]
> I guess this program is supposed to be installed for general usage? In that case please add a comprehensive help page (the we can use that to create a man-page) and make sure it gets installed to the right place.

I added some more info. Cool if you could comment on what you miss or what could be improved.

File: examples/co2brinepvt.cpp [hunk 1]
```diff
@@ -52,26 +52,41 @@ double densityBrine(const BrinePvt& brinePvt, const double p, const double T, co
...
+        help = help || (tmp  == "--h") || (tmp  == "--help");
+    }
+
+    if (argc < 5 || help) {
+        std::cout << "co2brinepvt computes PVT properties of a brine/co2 system " << std::endl;
+        std::cout << "for a given phase (oil or brine), pressure, temperature, salinity(optional), rs(optional) " << std::endl;
+        std::cout << "The properties support are: density, the inverse phase formation volume factor (invB), vicosity, " << std::endl;
+        std::cout << "saturated dissolution factor (rsSat) " << std::endl;
+        std::cout << "See CO2STORE in the OPM manual for more details." << std::endl;
+        std::cout << "USAGE:" << std::endl;
```
Comment (blattms) [inline examples/co2brinepvt.cpp:68]: Thanks.
Maybe we should put usage to one line and at the top?

Everywhere else we seem to use:
```
Usage: co2brinepvt <prop> <phase> <p> <T> <salinity> <rs>
... explanation
```

Comment (totto82) [reply examples/co2brinepvt.cpp:63]: updated

**Comment by totto82** [comment]
Do I need to do anything to get this small application part of the binaries in the next release?

**Comment by blattms** [comment]
I needed to double check.
Indeed, it seems you need to add to PROGRAM_SOURCE_FILES in addition
```
list (APPEND PROGRAM_SOURCE_FILES examples/co2brinepvt.cpp
```
then it will also get installed.

**Comment by totto82** [comment]
> Then it will also get installed, and be part of the binary packages.

Done. Thanks. Any reason not merging this now?

**Comment by blattms** [comment]
jankins build this please

File: CMakeLists_files.cmake [hunk 2]
```diff
@@ -44,7 +44,7 @@ list (APPEND EXAMPLE_SOURCE_FILES "/examples/co2brinepvt.cpp"
 
 # programs listed here will not only be compiled, but also marked for
 # installation
-list (APPEND PROGRAM_SOURCE_FILES)
+list (APPEND PROGRAM_SOURCE_FILES "/examples/co2brinepvt.cpp")
```
[the remark this answers is not in the document]
Comment (totto82) [reply CMakeLists_files.cmake:47]: both places or only here?

[the remark this answers is not in the document]
Comment (akva2) [reply CMakeLists_files.cmake:47]: both.

**Comment by blattms** [comment]
>+list (APPEND PROGRAM_SOURCE_FILES "/examples/co2brinepvt.cpp")
>
>remove leading /

Was my first thought, too. But on second thought: why should it hurt? Isn't Cmake
using full paths anyway?

**Comment by totto82** [comment]
jenkins build this please

**Comment by blattms** [comment]
jenkins build this please

**Comment by totto82** [comment]
@blattms Merge? I would like to have it in the release.

**Pull request merged**